/**/motion_executor_node:
  ros__parameters:

    update_rate: 30

    angular_goal_accuracy: 0.087 # rad
    angular_decel_distance: 0.785 # rad

    min_angular_speed: 2.5 # rad/s
    max_angular_speed: 3.5 # rad/s
    max_angular_accel: 10.0

    linear_goal_accuracy: 0.05 # m
    linear_decel_distance: 0.25 # m

    min_linear_speed: 0.3 # m/s
    max_linear_speed: 0.5 # m/s
    max_linear_accel: 2.0
